content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -135,6 +148,102 @@ $ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<t
 <title>Simple Bookstore App</title>
 {{< /text >}}
 
+{{< /tab >}}
+
+{{< tab name="Gateway API" category-value="gateway-api" >}}
+
+Create a [Kubernetes Gateway](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.Gateway),
+so you can access the bookinfo app from outside the cluster:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+metadata:
+  name: bookinfo-gateway
+  namespace: istio-system
+spec:
+  gatewayClassName: istio
+  listeners:
+  - name: http
+    port: 80
+    protocol: HTTP
+    allowedRoutes:
+      namespaces:
+        from: All
+EOF
+{{< /text >}}
+
+The Gateway is configured to listen for HTTP traffic on port 80 and allow
+[HTTPRoutes](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io%2fv1beta1.HTTPRoute) from any
+namespace to bind to the listener.
+
+Creating a Kubernetes Gateway resource deploys an associated
+[proxy service](/docs/tasks/traffic-management/ingress/gateway-api/#automated-deployment), so run the following command
+to wait for the gateway to be ready.
+
+If you are using a cluster that supports a [load balancer Service](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer),
+wait until the Gateway is ready:
+
+{{< text bash >}}
+$ kubectl wait --for=condition=ready gtw/bookinfo-gateway -n istio-system
+{{< /text >}}
+
+Otherwise, wait until the Gateway deployment is available:
+
+{{< text bash >}}
+$ kubectl wait --for=condition=available deploy/bookinfo-gateway-istio -n istio-system
+{{< /text >}}
+
+Route traffic through the Gateway to the `productpage` Service:
+
+{{< text bash >}}
+$ kubectl apply -f - <<EOF
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: HTTPRoute
+metadata:
+  name: productpage
+spec:
+  parentRefs:
+  - name: bookinfo-gateway
+    namespace: istio-system
+  hostnames:
+  - "bookinfo-gateway-istio.istio-system"
+  rules:
+  - matches:
+    - path:
+        type: PathPrefix
+        value: /productpage
+    backendRefs:
+    - name: productpage
+      port: 9080
+EOF
+{{< /text >}}
+
+You have now created an HTTPRoute configuration for the `productpage` service containing one route rule that allows
+traffic for path `/productpage`.
+
+Test your bookinfo application, it should work with or without the gateway:
+
+{{< text syntax=bash snip_id=verify_traffic_sleep_to_gateway >}}
+$ kubectl exec deploy/sleep -- curl -s http://bookinfo-gateway-istio.istio-system/productpage | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+{{< text bash >}}
+$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
+<title>Simple Bookstore App</title>
+{{< /text >}}
+
+{{< /tab >}}
+
+{{< /tabset >}}
```

Comment:
I would move these curl calls outside of the tab, and just set an env variable to the gateway host, which can then be used to access the gateway (Istio or k8s) in the rest of the doc.

```suggestion

{{< text bash >}}
$ export GATEWAY_URL=...
{{< /text >}}

{{< /tab >}}

{{< /tabset >}}

Test your bookinfo application, it should work with or without the gateway:

{{< text syntax=bash snip_id=verify_traffic_sleep_to_gateway >}}
$ kubectl exec deploy/sleep -- curl -s ${GATEWAY_URL}/productpage | grep -o "<title>.*</title>"
<title>Simple Bookstore App</title>
{{< /text >}}

{{< text bash >}}
$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
<title>Simple Bookstore App</title>
{{< /text >}}

{{< text bash >}}
$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | grep -o "<title>.*</title>"
<title>Simple Bookstore App</title>
{{< /text >}}
```